public/app/plugins/datasource/jaeger/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@grafana-plugins/jaeger",
+  "description": "Jaeger plugin for Grafana",
+  "private": true,
+  "version": "10.4.0-pre",
+  "dependencies": {
+    "@emotion/css": "11.11.2",
+    "@grafana/data": "workspace:*",
+    "@grafana/experimental": "1.7.8",
+    "@grafana/o11y-ds-frontend": "workspace:*",
+    "@grafana/runtime": "workspace:*",
+    "@grafana/ui": "workspace:*",
+    "@reduxjs/toolkit": "1.9.5",
+    "buffer": "6.0.3",
+    "events": "3.3.0",
```

Comment:
As mentioned futher down I don't believe these node polyfills are necessary for this plugin to function.